TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java

```diff
@@ -26,7 +26,7 @@
  */
 public class InsertData extends AppCompatActivity {
 
-    String id,name,poojaTyp,overall,money,paidCheck = "NOT PAID";
+    String id,name,poojaTyp,overall,money,paidCheck = getString(R.string.NOT_PAID);
```

Comment:
here you're accessing resources too early You can only use your activity as a Context with Resources in onCreate() or later in the activity lifecycle
so it producing null pointer exception please change it

Reply:
@adarshsiva please revert this change.

Reply:
done